### DISCUSSION TITLE
Set default folder for Assets

### DISCUSSION THREAD

**Discussion by fuchsbox** [statamic/cms#8420] [General]
Where can I set the default folder for assets, that opens when I click the „Browse“ Button? Thank you! 😃
<img width="523" alt="2023-07-08_11-35-56" src="https://github.com/statamic/cms/assets/118173137/f163320f-c7f9-4443-8013-b752f022adf3">

**Comment by jackmcdade** [comment]
I’m that fieldtype’s blueprint settings. On Jul 8, 2023, at 5:37 AM, Philip Fuchslocher ***@***.***> wrote:﻿
Where can I set the default folder for assets, that opens when I click the „Browse“ Button? Thank you! 😃
